benchexec/tools/template.py

```diff
@@ -151,6 +151,24 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         """
         return result.RESULT_UNKNOWN
 
+    def determine_result_for_property(self, returncode, returnsignal, output, isTimeout, property):
+        """
+        Parse the output of the tool and extract the verification result for specified property.
+        This method always needs to be overridden for multi-property verification.
+        If the tool gave a result, this method needs to return one of the
+        benchexec.result.RESULT_* strings.
+        Otherwise an arbitrary string can be returned that will be shown to the user
+        and should give some indication of the failure reason
+        (e.g., "CRASH", "OUT_OF_MEMORY", etc.).
+        @param returncode: the exit code of the program, 0 if the program was killed
+        @param returnsignal: the signal that killed the program, 0 if program exited itself
+        @param output: a list of strings of output lines of the tool (both stdout and stderr)
+        @param isTimeout: whether the result is a timeout
+        (useful to distinguish between program killed because of error and timeout)
+        @param property: property, for which status is expected
+        @return a non-empty string, usually one of the benchexec.result.RESULT_* constants
```

Comment:
It was suggested to use the following format for correct results and results.xml file: 
`propertyK: true|false|unknown`
For example:
`unreach-call(__VERIFIER_error_linux_usb_urb): false`
If we use `RESULT_FALSE_REACH`, which is used in CPAchecker by default, there will be duplication:
`unreach-call(__VERIFIER_error_linux_usb_urb): false(unreach-call)`
For other properties RESULT_FALSE_* constants contain full property name, for example:
`valid-deref: false(valid-deref)`
What do you think about such duplications (in property names and in RESULT_FALSE_* constants)?